src/P/EitherT.hs

```diff
@@ -11,3 +13,6 @@ firstEitherT f = bimapEitherT f id
 
 secondEitherT :: (Functor m) => (a -> b) -> EitherT e m a -> EitherT e m b
 secondEitherT f = bimapEitherT id f
+
+eitherTFromMaybe :: Functor m => b -> m (Maybe a) -> EitherT b m a
+eitherTFromMaybe l r = EitherT $ maybe (Left l) pure <$> r
```

Comment:
`transformers` is already a dependency on `either` so `p` already depends on it, but nvm, i think this is already too much thinking on the issue, its no longer worth the trouble, the code is fine :)